docs/running/notebooks/01_running_program.ipynb

```diff
@@ -88,29 +87,18 @@
    "source": [
     "After importing the necessary classes and configuring them, we can run the program by calling the `run()` method of the `QuantumServerless` object:\n",
     "\n",
-    "`Program` accepts couple of required parameters:\n",
+    "`Program` accepts a few parameters:\n",
     "- title - name of the program\n",
     "- entrypoint - name of python file you want to execute\n",
     "- working_dir - folder where  your script is located. This is optional parameter and will be current folder by default. "
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 12,
+   "execution_count": 3,
    "id": "d51df836-3f22-467c-b637-5803145d5d8a",
    "metadata": {},
-   "outputs": [
-    {
-     "data": {
-      "text/plain": [
-       "<Job | 3bbb5ced-06d7-44ff-a08c-58266881d54c>"
-      ]
-     },
-     "execution_count": 12,
-     "metadata": {},
-     "output_type": "execute_result"
-    }
-   ],
+   "outputs": [],
```

Comment:
this is more a general comment: how should we store the notebooks? with no output, or with some desired output? (I think I'd lean towards no output, but curious what others think)